nat-lab/tests/test_multicast_connection.py

```diff
@@ -52,6 +52,22 @@ def generate_setup_parameter_pair(
         ]),
         "mdns",
     ),
+    pytest.param(
+        generate_setup_parameter_pair([
+            (ConnectionTag.MAC_VM, AdapterType.BoringTun),
+            (ConnectionTag.DOCKER_CONE_CLIENT_1, AdapterType.BoringTun),
+        ]),
+        "ssdp",
+        marks=pytest.mark.mac,
+    ),
+    pytest.param(
+        generate_setup_parameter_pair([
+            (ConnectionTag.DOCKER_CONE_CLIENT_1, AdapterType.BoringTun),
+            (ConnectionTag.MAC_VM, AdapterType.BoringTun),
+        ]),
+        "mdns",
+        marks=pytest.mark.mac,
+    ),
```

Comment:
Why is there no linux native adapter in this whole list?

Reply:
I only adapted whatever was there for macOS, if it's just a another parameter pair, I can add it here.

Reply:
I'm not asking you to add it in this task, I'm asking out of curiosity and because quick skim of the RFC-0047 didn't answer my question.